src/components/SelectionList/BaseSelectionList.js

```diff
@@ -80,10 +94,17 @@ function BaseSelectionListRadio(props) {
         // because React Native accounts for it in getItemLayout
         itemLayouts.push({length: 0, offset});
 
+        if (selectedCount > 1 && !props.canSelectMultiple) {
+            throw new Error(
```

Comment:
Throwing an `Error` would lead to a crash in the app, wouldn't? I wonder if this is really necessary..

Reply:
Correct, I wanted to make it very clear for the dev when developing, but I guess a `Log.warn` serves the same purpose and is less risky 👍🏻